nextcord/ext/commands/errors.py

```diff
@@ -859,6 +859,14 @@ class NoEntryPointError(ExtensionError):
     def __init__(self, name: str) -> None:
         super().__init__(f"Extension {name!r} has no 'setup' function.", name=name)
 
+class InvalidSetupArguments(ExtensionError):
+    """An exception raised when an extension contains a ``setup`` function which does except ``kwargs`` but ``kwargs`` were passed.
```

Comment:
Is it at all possible to wrap this to two lines as its 132 characters long? Simply wrapping it in the middle should work.